### PR TITLE
Submit

### PR DESCRIPTION
**Pull request by DrPandemic**
- Open a socket to receive blocks
- Submit the block to the bitcoin network

### CODE REVIEW

File: server/src/bitcoin_helper.ts [hunk 1]
```diff
@@ -7,9 +7,10 @@ import * as Client from "bitcoin-core";
 export function connectToBC(): any {
   return new Client(
     {
-      network: "testnet",
-      username: creds.username,
-      password: creds.password
+      network: "regtest",
+      username: "username",
+      password: "password",
```
Comment (JesseEmond) [inline server/src/bitcoin_helper.ts:12]: Is the removal of `creds` on purpose? If so, what's the rationale?

File: server/src/app.ts [hunk 2]
```diff
@@ -4,17 +4,20 @@
...
-  sock.connect(ADDRESS);
-  sock.subscribe("");
-  sock.on('message', function(data:any) {
-    console.log(data.toString());
-  });
+  let socket = zmq.socket("push");
+  socket.connect(HOST + (PORT + 1));
+
+  setInterval(function() {
+    var value = Math.floor(Math.random()*100);
```
Comment (JesseEmond) [inline server/src/app.ts]: what is this value for?

File: bitcoin-config/bitcoin3.conf [hunk 3]
```diff
@@ -6,6 +6,9 @@ datadir=./data3
 port=18333
 daemon=1
 rpcport=16593
+server=1
+rpcuser=username
+rpcpassword=password
```
Comment (JesseEmond) [inline bitcoin-config/bitcoin3.conf:11]: just out of curiosity: why was this moved from `bitcoin2` to `bitcoin3`?

File: server/src/bitcoin_helper.ts (hunk 1 above)
Comment (DrPandemic) [inline server/src/bitcoin_helper.ts:12]: I don't care if someone use my wallet, since it's on regtest

File: server/src/app.ts (hunk 2 above)
Comment (DrPandemic) [inline server/src/app.ts]: test, will be removed

File: server/src/bitcoin_helper.ts (hunk 1 above)
Comment (JesseEmond) [inline server/src/bitcoin_helper.ts:12]: Ok so we'll just use `regtest` from now on?

File: bitcoin-config/bitcoin3.conf (hunk 3 above)
Comment (DrPandemic) [inline bitcoin-config/bitcoin3.conf:11]: Mhh, why not. No good reason, I was playing with the setup

File: server/src/bitcoin_helper.ts (hunk 1 above)
Comment (DrPandemic) [inline server/src/bitcoin_helper.ts:12]: Hmmmm, I guess?

**Comment by JesseEmond** [comment]
LGTM! :+1: Merging.
